src/app/nav-menu/nav-menu.component.ts

```diff
@@ -28,9 +29,7 @@ export class NavMenuComponent implements OnInit {
   }
 
   private admin(): void {
-    const content: NgbModalRef = this.modalService.open(ModalContentComponent);
-    content.componentInstance.title = 'You\'re an Admin!';
-    content.componentInstance.message = 'Look at you, being an admin.';
+    this.router.navigate(['admin']);
```

Comment:
Instead of doing this, you can put `routerLink="/admin"` on the html tag instead of `(click)="admin()"`.